ui/app/templates/components/secret-edit.hbs

```diff
@@ -53,43 +53,85 @@
     <label for="json" class="has-text-grey">JSON</label>
   </div>
   {{#if (and (eq mode 'show') (or canEditV2Secret canEdit))}}
-    <div class="control">
       {{#let (concat 'vault.cluster.secrets.backend.' (if (eq mode 'show') 'edit' 'show')) as |targetRoute|}}
-        {{#if isV2}}
-          <LinkTo
-            @params={{array targetRoute model.id (query-params version=this.modelForData.version)}}
-            @replace={{true}}
-            class="link link-plain has-text-weight-semibold"
-          >
-            Create new version
-          </LinkTo>
-        {{else}}
-          <LinkTo
-            @params={{array targetRoute model.id}}
-            @replace={{true}}
-            class="link link-plain has-text-weight-semibold"
+        <div class="control">
```

Comment:
We'll want to check to make sure that if it's v2 that the version isn't deleted or destroyed before we render this dropdown.